files/ru/web/css/backdrop-filter/index.md

```diff
@@ -75,7 +75,7 @@ body {
 }
 
 body {
-  background-image: url('https://lorempixel.com/400/200/');
+  background-image: url('https://yari-demos.prod.mdn.mozit.cloud/en-US/docs/Web/CSS/backdrop-filter/anemones.jpg');
```

Comment:
@placidp please upload `anemones.jpg` to `files/ru/web/css/backdrop-filter/` directory to have an ability to use relative path.

Reply:
> @placidp please upload `anemones.jpg` to `files/ru/web/css/backdrop-filter/` directory to have an ability to use relative path.

Thanks for your response! Please take a look at my changes.